### PR TITLE
IDEMPIERE-5152 : Add title on broadcast messages

### PR DESCRIPTION
**Pull request by nmicoud**
I also set AD_BroadcastMessage_Trl.BroadcastMessage IsAlwaysUpdateable to 'N' to make it consistent with header

### CODE REVIEW

**Comment by nmicoud** [comment]
I also remove "width: 370px; " from the style of divAlign so the message can be displayed using all the width (in my tests, seems ok)

File: org.adempiere.base/src/org/compiere/model/X_AD_BroadcastMessage.java [hunk 1]
```diff
@@ -47,6 +47,20 @@ public X_AD_BroadcastMessage (Properties ctx, int AD_BroadcastMessage_ID, String
...
+      /** if (AD_BroadcastMessage_ID == 0)
+        {
+			setAD_BroadcastMessage_ID (0);
+			setBroadcastFrequency (null);
+			setBroadcastMessage (null);
+			setBroadcastType (null);
+			setTarget (null);
+        } */
+    }
+
```
Comment (hengsin) [inline org.adempiere.base/src/org/compiere/model/X_AD_BroadcastMessage.java:63]: as it is, if your changes is for release_9 branch, you have to re-generate model using release_9 branch. The model class generated from master branch is not compatible with release_9 source.

Comment (nmicoud) [reply org.adempiere.base/src/org/compiere/model/X_AD_BroadcastMessage.java]: done

**Comment by CarlosRuiz-globalqss** [comment]
> I also remove "width: 370px; " from the style of divAlign so the message can be displayed using all the width (in my tests, seems ok)

did you test it in mobile?

**Comment by nmicoud** [comment]
just now ; using Firefox developer tools, seems ok to me

**Comment by CarlosRuiz-globalqss** [comment]
> just now ; using Firefox developer tools, seems ok to me

I just did a test here:
https://test-feature.idempiere.org:3243/webui/index.zul

You can try logging in with GardenAdmin user.
The window is not really usable in mobile - it doesn't show the full message, it cannot be resized, and it cannot be acknowledged  (I suspect this was not introduced by your changes)

**Comment by nmicoud** [comment]
ok, so let's remove this change from the ticket

I think there are other improvements for the style (at the moment, vertical-align: middle; text-align: center; is hardcoded ; that should be better to be able to manage that per message)

**Comment by CarlosRuiz-globalqss** [comment]
> ok, so let's remove this change from the ticket

tested in test.idempiere.org - is not caused by you, and indeed it seems your change made it slightly better, but still failing

**Comment by nmicoud** [comment]
so let's keep it :)
Anyway, wouldn't be better to be able to manage the whole style from AD_BroadcastMessage ?

File: org.adempiere.base/src/org/compiere/model/X_AD_BroadcastMessage.java (hunk 1 above)
Comment (CarlosRuiz-globalqss) [reply org.adempiere.base/src/org/compiere/model/X_AD_BroadcastMessage.java]: > as it is, if your changes is for release_9 branch, you have to re-generate model using release_9 branch. The model class generated from master branch is not compatible with release_9 source.

This is going to do the maintenance of release-9 and master too difficult - I think better we are needing to integrate the changes on IDEMPIERE-5126

**Comment by nmicoud** [comment]
done
Ability to move the panel is a good thing !
Thanks

**Review by CarlosRuiz-globalqss — APPROVED** [review]

**Pull request merged**